cluster/examples/kubernetes/ceph/filesystem.yaml

```diff
@@ -13,13 +13,13 @@ spec:
   # The metadata pool spec. Must use replication.
   metadataPool:
     replicated:
-      size: 3
+      size: 6
```

Comment:
please revert the test changes to this yaml

Reply:
Sure.

Reply:
Done!, please review now, I have fixed all this.
thanks